PowerShell/BuildingBlocks/CustomerAndResources/AssignSubscriptions.ps1

```diff
@@ -0,0 +1,22 @@
+﻿# Initialize the context
+.\Init.ps1 -Customer
+
+# Get the end users under that customer (this example retrieves the top 100)
+$customerEndUsers =  Get-BT_CustomerEndUser -Ticket $mspc.CustomerTicket -OrganizationId $mspc.Customer.OrganizationId -IsDeleted $False -PageSize 100
```

Comment:
One about parameters here, as this may form a template for many more scripts to come. Are the following parameters really necessary or are they provided more for clarity?

1) Is filter on OrganizationId strictly required? I thought if 'CustomerTicket' is used, the appropriate filtering should already be applied.
2) Same for IsDeleted flag - isn't '-IsDeleted $false' already a default?

Reply:
Removed the filter on OrganizationId.
The filter on IsDeleted is needed though, we don't add it by default.